server/__tests__/api.test.js

```diff
@@ -1,10 +1,30 @@
 import request from 'supertest';
 import app from '../app';
 
+jest.mock('@google-cloud/bigquery');
+
+/**
+ * Test suite for the health check endpoint (/api)
+ */
 describe('Health Check', () => {
   it('Should be healthy', async () => {
     const res = await request(app).get('/api');
     expect(res.statusCode).toEqual(200);
     expect(res.text).toBe('API working');
   });
 });
+
+/**
+ * Test suite for mixed-content endpoint (api/mixed-content)
+ */
+describe('mixed-content', () => {
+  it('Check /top-websites-with-mixed-content', async () => {
+    const res = await request(app).get('/api/mixed-content/'+
+    'top-websites-with-mixed-content');
+
+    expect(res.body.description).toBe('Top websites with most mixed content'+
```

Comment:
There is a missing space here that is making the tests to fail.

Just fix it and we're ready to go

Reply:
Done!